### ISSUE TITLE
Adjust volume of individual sounds

### ISSUE THREAD

**Issue by ccidral** [tomighty/tomighty#11]
Let the user configure the volume of each individual sound in the options dialog using a slider control.

**Comment by lecterror** [comment]
+1 to this, it is highly unpleasant when tomighty dings as loud as Fear Factory song you're currently listening! >_<

**Comment by donovankeith** [comment]
Yes please! The ticking sound is nice, but it's a bit loud overall for me - and I could stand to have the alarms a bit louder. In addition to individual adjustments, an overall volume adjustment for the app would be great.

**Comment by laviyatziv** [comment]
+1

**Comment by letalvoj** [comment]
:+1:

**Comment by AleksMeshkov** [comment]
any updates?
